tests/rustdoc-json/attrs/target_feature.rs

```diff
@@ -0,0 +1,3 @@
+//@ is "$.index[?(@.name=='test')].attrs" '["#[target_feature(enable=\"avx\")]"]'
+#[target_feature(enable = "avx")]
+pub fn test() {}
```

Comment:
If rustdoc is going to contain normalization logic, this should test all the cases in https://github.com/rust-lang/rust/pull/142876#issuecomment-2994277954